kong/db/schema/entities/routes.lua

```diff
@@ -126,7 +126,7 @@ else
         field_sources = { "path_handling" },
         fn = function(entity)
           if entity.path_handling == "v1" and kong.configuration.router_flavor ~= "traditional" then
-            return nil, "path_handling = 'v1' is deprecated and not supported with router_flavor = 'traditional_compatible'"
+            return nil, "path_handling = 'v1' is deprecated and not supported when router_flavor is not 'traditional'"
```

Comment:
How about `only supported when router_flavor is 'traditional'`

Reply:
done